src/lib/i18n/de.ts

```diff
@@ -0,0 +1,161 @@
+export default {
+  translation: {
+    //General
+    'general.error': 'Fehler',
+    'general.cancel': 'Abbrechen',
+    'general.attachments': 'Anhänge',
+    'general.trash': 'Papierkorb',
+    'general.allnote': 'Alle Notizen',
+    'general.signin': 'Anmleden',
+    'general.signOut': 'Abmelden',
+    'general.save': 'Speichern',
+    'general.default': 'Standard',
+    'general.networkError': 'Netzwerk-Fehler',
+
+    // Storage
+    'storage.name': 'Speicher Name',
+    'storage.noStorage': 'Keine Speicher',
+    'storage.create': 'Speicher erstellen',
+    'storage.edit': 'Speicher ändern',
+    'storage.rename': 'Speicher umbenennen',
+    'storage.renameMessage': 'Name für Speicher ändern',
+    'storage.remove': 'Speicher entfernen',
+    'storage.removeMessage': 'Speicher wird von App entknüpft',
+    'storage.delete': 'Speicher löschen',
+    'storage.move': 'Notiz verschieben',
+    'storage.moveTitle': 'Notiz in anderen Speicher verschieben',
+    'storage.moveMessage':
+      'Es wird versucht eine Notiz in einen anderen Speicher zu verschieben.',
+    'storage.copy': 'Notiz kopieren',
+    'storage.typeLocal': 'Lokal',
+    'storage.typeCloud': 'Cloud',
+    'storage.needSignIn': 'Du musst dich anmelden um einen Cloud-Speicher zu erstellen.',
+    'storage.syncDate': 'Zuletzt synchronisiert am',
+
+    //Folder
+    'folder.create': 'Neuer Ordner',
+    'folder.rename': 'Ordner umbenennen',
+    'folder.renameMessage':
+      'Gib den neuen Ordnernamen an, jede Notiz und Unterordner-Pfad wird aktualisiert.',
+    'folder.renameErrorMessage': 'Ordner konnte nicht umbenannt werden',
+    'folder.remove': 'Ordner löschen',
+    'folder.removeMessage': 'Alle Notizen und Unterordner werden gelöscht.',
+
+    //Tag
+    'tag.tag': 'Etikett',
+    'tag.remove': 'Etikett löschen',
+    'tag.removeMessage': 'Das Etikett wird von allen Notizen entfernt.',
+
+    //Note
+    'note.duplicate': 'Duplizieren',
+    'note.delete': 'Löschen',
+    'note.delete2': 'Notiz löschen',
+    'note.deleteMessage': 'Die Notiz wird endgültig gelöscht',
+    'note.empty': 'Leere Notiz',
+    'note.unselect': 'Keine Notiz ausgewählt',
+    'note.search': 'Notizen durchsuchen',
+    'note.nothing': 'Keine Notizen',
+    'note.nothingMessage': 'Keine Notizen gefunden.',
+    'note.noTitle': 'Kein Titel',
+    'note.date': 'her',
```

Comment:
my suggestion: Datum

Reply:
'note.date'='ago' in the english  which should be translated to 'her', or is this a mistake in the english translation file?

Reply:
my bad, I wasn't aware of the context.  You are correct.